addons/crm/models/crm_lead.py

```diff
@@ -107,8 +107,14 @@ class Lead(models.Model):
         help="Classify and analyze your lead/opportunity categories like: Training, Service")
     color = fields.Integer('Color Index', default=0)
     # Opportunity specific
-    planned_revenue = fields.Monetary('Expected Revenue', currency_field='company_currency', tracking=True)
+    planned_revenue = fields.Monetary('Expected Revenues', currency_field='company_currency', tracking=True)
     expected_revenue = fields.Monetary('Prorated Revenue', currency_field='company_currency', store=True, compute="_compute_expected_revenue")
+    expected_monthly_revenue = fields.Monetary('Expected MRR', currency_field='company_currency', store=True,
+                                               compute="_compute_expected_monthly_revenue", groups="crm.group_use_recurring_revenues")
+    prorated_monthly_revenue = fields.Monetary('Prorated MRR', currency_field='company_currency', store=True,
```

Comment:
Wow, what a soup of inverted naming (not your fault here! :) 
planned_revenue = 'Expected Revenue'
expected_revenue = 'Prorated Revenue' 
"Historical Dafuck"
Anyways, this field is never used. I think we should add it in the dashboard view (same as for prorated_revenue_aggregate brol)
We'll see if we need to align to old messy naming brol or if if we take the opportunity to clean original fields naming.. or we we do nothing and let this as is.. Wait and see !

Reply:
So, so we will take the opportunity to rename the old fields (watch out migration stuffs)
expected_revenue becomes prorated_revenue (the string does not change)
planned_revenue become expected_revenue ((the string does not change)
And you can keep your new fields as is :) 
cc : @tde-banana-odoo @LNAodoo